### DISCUSSION TITLE
The RustDesk Uninstaller doesn't delete the %HOMEPATH%\AppData\Local\rustdesk and %HOMEPATH%\AppData\Roaming\RustDesk directories

### DISCUSSION THREAD

**Discussion by x011** [rustdesk/rustdesk#7064] [Feature Request]
### Bug Description

Directories %HOMEPATH%\AppData\Local\rustdesk and %HOMEPATH%\AppData\Roaming\RustDesk should be deleted on uninstall

### How to Reproduce

Uninstall RustDesk

### Expected Behavior

Directories %HOMEPATH%\AppData\Local\rustdesk and %HOMEPATH%\AppData\Roaming\RustDesk should be deleted

### Operating system(s) on local side and remote side

Windows 11 -> Ubuntu 22

### RustDesk Version(s) on local side and remote side

1.2.3  -> 1.1.10-3

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/4313821/df3cfc22-822e-4e30-af9f-b6eefdf24d9d)
![image](https://github.com/rustdesk/rustdesk/assets/4313821/85f55e70-5f69-45f1-9851-56e361bda63c)

**Comment by rustdesk** [comment]
@21pages let's add a dialog to tell user if removing configuration files and temp files. No high priority.
